src/sentry/models/commitfilechange.py

```diff
@@ -12,7 +12,7 @@ class CommitFileChange(Model):
 
     organization_id = BoundedPositiveIntegerField(db_index=True)
     commit = FlexibleForeignKey('sentry.Commit')
-    filename = models.TextField()
+    filename = models.CharField(max_length=255)
```

Comment:
Did you intend to include this too?

Reply:
Yeah. The key changes from the previous commit were for EmailField which has a new default length.